java/doc/api.rst

```diff
@@ -7,51 +7,45 @@ Basic API
 ``Ray.init()``
 ~~~~~~~~~~~~~~
 
-Ray.init should be invoked before any other Ray functions to initialize
+``Ray.init`` should be invoked before any other Ray functions to initialize
 the runtime.
 
 ``@RayRemote``
 ~~~~~~~~~~~~~~
 
 The ``@RayRemote`` annotation can be used to decorate static java
 methods and classes. The former indicates that a target function is a remote
-function, which is valid with the follow requirements.
+function, which is valid with the following requirements:
 
 - It must be a public static method.
-- Its parameters and return value cannot be primitive types like int or
-  double but can use wrapper classes like Integer or Double.
 - The method must be deterministic for task reconstruction to behave correctly.
 
 When the annotation is used for a class, the class becomes an actor class
 (an encapsulation of state shared among many remote functions). The
 member functions can be invoked using ``Ray.call``. The requirements for
-an actor class are as follows.
+an actor class are as follows:
 
 - It must have a constructor without any parameters.
 - Any inner class must be public static.
-- It must not have a member field or method decorated using ``public static``,
-  as the semantic is undefined with multiple instances of this same class on
-  different machines
-- An actor method must be decorated using ``public`` but not ``static``
-- The other requirements are the same as for remote functions.
+- It must not have any static fields or methods, as the semantic is undefined
+  with multiple instances of this same class on different machines.
+- An actor method must be ``public``.
```

Comment:
Do all methods of the actor need to be public or just the ones that can be invoked remotely?

Reply:
Only the ones that can be invoked remotely. I'll make it more clear.